Python/chapter01/1.1 - Is Unique/jesus_1.1.sol.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
Would it be possible for you to add a module docstring with the problem statement? Not all (volunteer!) reviewers have a copy of the book.